pypika/tests/test_selects.py

```diff
@@ -355,6 +355,18 @@ def test_where_field_equals_for_update(self):
         q = Query.from_(self.t).select("*").where(self.t.foo == self.t.bar).for_update()
         self.assertEqual('SELECT * FROM "abc" WHERE "foo"="bar" FOR UPDATE', str(q))
 
+    def test_where_field_equals_for_update_nowait(self):
+        q = Query.from_(self.t).select("*").where(self.t.foo == self.t.bar).for_update(nowait=True)
+        self.assertEqual('SELECT * FROM "abc" WHERE "foo"="bar" FOR UPDATE NOWAIT', str(q))
+
+    def test_where_field_equals_for_update_skip_locked(self):
+        q = Query.from_(self.t).select("*").where(self.t.foo == self.t.bar).for_update(skip_locked=True)
+        self.assertEqual('SELECT * FROM "abc" WHERE "foo"="bar" FOR UPDATE SKIP LOCKED', str(q))
+
+    def test_where_field_equals_for_update_of(self):
```

Comment:
Can you please add a test with multi of values?

Reply:
OK, I will do that

Reply:
Oh, sorry for not being clear enough. I meant a tuple for of kwarg with more than one value. That said, please keep the newly added tests too. It's good to have them nevertheless.

Reply:
Sorry my English is not so good, could you give a example?

Reply:
I mean to add a test in which "of" kwarg has a tuple with more than one value also. Like in .for_update(of=("abc","def")).

Reply:
OK